### ISSUE TITLE
Support APNG

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [CodeDead/AniView#1]
The Animated Portable Network Graphics ([APNG](https://en.wikipedia.org/wiki/APNG)) file format is an extension to PNG with animate functonality, it works similarly to animated GIF files, while supporting 24-bit images and 8-bit transparency not available for GIFs.

**Comment by CodeDead** [comment]
Thank you for letting us know. We'll look into it and get back to you!

**Comment by KrasnayaPloshchad** [comment]
Yeah, since Chrome supported this format, it would be awesome for you.
